XRTK-Core/Packages/com.xrtk.core/Definitions/Utilities/MixedRealityPose.cs

```diff
@@ -122,7 +122,7 @@ bool IEqualityComparer.Equals(object left, object right)
         public bool Equals(MixedRealityPose other)
         {
             return Position == other.Position &&
```

Comment:
Why not also use Vector2.Equals?

Reply:
Because the equality operator is enough here.

The equality operator on the quaternion is prone to float precision errors, where as calling `Equals` outright does not.

This change was ported from https://github.com/microsoft/MixedRealityToolkit-Unity/pull/4309